src/cljs/harja/ui/yleiset.cljs

```diff
@@ -532,15 +532,18 @@ lisätään eri kokoluokka jokaiselle mäpissä mainitulle koolle."
                  (partition 2 otsikot-ja-arvot))])
 
 (defn- luo-haitarin-rivi [piiloita? rivi]
-  ^{:key (:otsikko @rivi)}
-  [:div.haitari-rivi
-   [:div.haitari-heading.klikattava
-    {:on-click #(do
-                  (swap! rivi assoc :auki (not (:auki @rivi)))
-                  (.preventDefault %))}
-    [:span.haitarin-tila (if (:auki @rivi) (ikonit/livicon-chevron-down) (ikonit/livicon-chevron-right))]
-    [:div.haitari-title (when piiloita? {:class "haitari-piilossa"}) (or (:otsikko @rivi) "")]]
-   [:div.haitari-sisalto (if (:auki @rivi) {:class "haitari-auki"} {:class "haitari-kiinni"}) (:sisalto @rivi)]])
+  (let [avaa-tai-sulje (fn [event]
```

Comment:
Oisko mittään, jos avaa-tai-sulje funktion nimeäsi aavistuksen tarkemmin. Tyyliin avaa-tai-sulje-rivi ? Tuleeko liian pitkä jos ois avaa-tai-sulje-haitari-rivi tai toggle-auki-haitari-rivi ?

Reply:
Olisi hyvä kyllä :) Muutan vielä

Reply:
Käytin nyt näitä kaikkia: avaa-ja-sulje-rivi, avaa-ja-sulje-haitari ja avaa-ja-sulje-ryhma tilanteen mukaan. Ehtisitkö vielä katsoa nämä?